packages/core/src/login/webview/vue/amazonq/backend_amazonq.ts

```diff
@@ -56,7 +56,13 @@ export class AmazonQLoginWebview extends CommonAuthWebview {
                 isReAuth: false,
             })
 
-            await awsIdSignIn()
+            const conn = await awsIdSignIn()
+            const registration = await conn.getRegistration()
+            this.storeMetricMetadata({
```

Comment:
Does this make sense to do in `AuthUtil` instead? Under the hood this eventuall calls `AuthUtil.connectToAwsBuilderId`.

If true, then the same would be for `connectToEnterpriseSso`